dev/TreeView/TreeViewItem.cpp

```diff
@@ -235,7 +235,7 @@ void TreeViewItem::OnDragEnter(winrt::DragEventArgs const& e)
 
 void TreeViewItem::OnDragLeave(winrt::DragEventArgs const& e)
 {
-    winrt::DragEventArgs args = e;
+    const winrt::DragEventArgs& args = e;
     if (auto treeView = AncestorTreeView())
```

Comment:
I don't mind this rewrite *but* we should probably just rewrite these functions to not have a local because the alias doesn't add anything meaningful. That said also -- would clang tidy be ok with "auto &&args = e" too? Or would it want to rewrite that as well?

Reply:
I can remove this occurance. args does not seem to be even used.